src/eventHandlers/checkSuite/index.spec.ts

```diff
@@ -16,7 +16,7 @@ const COMMIT_HEADLINE = 'Update test';
 
 const octokit = new GitHub('SECRET_GITHUB_TOKEN');
 
-jest.spyOn(core, 'info').mockImplementation();
+const infoSpy = jest.spyOn(core, 'info').mockImplementation();
 
 const warningSpy = jest.spyOn(core, 'warning').mockImplementation();
```

Comment:
Do we need a newline here?

Reply:
fixed with https://github.com/ridedott/dependabot-auto-merge-action/pull/233/commits/8fa422481dda410cd7bc52b6c04ead5cbcab9c54